src/main/java/bhs/devilbotz/commands/ArmUp.java

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package bhs.devilbotz.commands;
+
+import bhs.devilbotz.subsystems.Arm;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class ArmUp extends CommandBase {
+  /** Creates a new ArmUp. */
+  private Arm _arm;
```

Comment:
Remove _ in variable:
eg:
arm instead of _arm

(update the rest of the references)